<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateJobTicket</class>
 <widget class="QDialog" name="GenerateJobTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Job Ticket</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="txtProductDetails">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>171</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Order Details</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="spinOrderQty">
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>LPO No.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="drpOrderDate">
          <property name="displayFormat">
           <string>dd MMM yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Required Date:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Order Date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtLPONo"/>
        </item>
        <item row="2" column="1">
         <widget class="QDateEdit" name="dtpRequiredDate">
          <property name="displayFormat">
           <string>dd MMM yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Order Qty:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Total Price:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="txtOrderTotal">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Production Details</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="txtPacking">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="txtHDPE">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Inks:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Master Batch:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="txtMasterBatch">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="txtInks">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Packing Breakdown:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>HDPE:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Packing:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QTableWidget" name="tblPackingBreakdown">
           <property name="minimumSize">
            <size>
             <width>390</width>
             <height>121</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>390</width>
             <height>121</height>
            </size>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::DashLine</enum>
           </property>
           <row>
            <property name="text">
             <string>Full Packs:</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Partial Packs:</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Total:</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>No. Of Packs</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Caps Per Pack:</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Total Caps:</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Finishing:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="txtFinishing">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Sales Details</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="tblSalesDetails">
           <property name="minimumSize">
            <size>
             <width>465</width>
             <height>148</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>465</width>
             <height>148</height>
            </size>
           </property>
           <row>
            <property name="text">
             <string>Bottles:</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Closures:</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Qty</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Price/Pc</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Total Price</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCommandLinkButton" name="cmdGenerateJobTicket">
       <property name="text">
        <string>Generate Job Ticket</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateJobTicket</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
